BuildAndTest.cmd

```diff
@@ -19,18 +19,20 @@ if "%ERRORLEVEL%" NEQ "0" (
 )
 popd
 
-
 pushd .\src\security_utilities_rust_ffi\
 call cargo clean --release
-call cargo build --release
+call cargo build --release --target x86_64-pc-windows-msvc
+call cargo build --release --target i686-pc-windows-msvc
```

Comment:
>windows [](http://example.com/codeflow?start=44&length=7)

why no linux support? is this coming later?

Reply:
@michaelcfanning That'll be a follow-on to this PR